components/server/src/projects/projects-service.ts

```diff
@@ -467,7 +468,10 @@ export class ProjectsService {
 
             return project;
         } catch (error) {
-            log.error("Prebuild settings migration failed.", { projectId: project.id, error });
+            log.error("Prebuild settings migration failed: " + error, {
```

Comment:
`log.error("Prebuild settings migration failed:", error, { projectId: project.id });`

Reply:
we don't scrub errors, I don't think you need trusted value around it